doc/pages/tips.adoc

```diff
@@ -112,3 +119,25 @@ aug MyClojureSetting
   au FileType clojure nnoremap <buffer> <Leader>Go :<C-u>IcedEval (user/reset)<CR>
 aug END
 ----
+
+=== Miscellaneous
+
+By default https://github.com/guns/vim-sexp[vim-sexp]'s paredit mode is
+enabled.  If, for example, you don't want typing '(' to insert a pair of parentheses, 
+you can turn this off by disabling vim-sexp's key mappings, and
+possibly adding back vim-sexp key mappings that you want to keep.  This
+won't affect vim-iced.
+
+To disable vim-sexp's key mappings, put a line like `let
+g:sexp_filetypes=''` in a startup file such as your vimrc.
+(`g:sexp_filetypes` is normally set to "clojure,scheme,lisp,timl",
+which are the filetypes for which vim-sexp's mappings are normally
+enabled.)  
+
+To re-enable some of vim-sexp's key mappings, follow the instructions at
+`:help sexp-explicit-mappings` to define a function like
+`s:vim_sexp_mappings()` that is listed there.  For example, to keep all vim-sexp
+mappings except the the ones that cause matching delimiters to be
+inserted automatically, delete the last several lines from the the
+definition of `s:vim_sexp_mappings()`.  Then follow the instructions
+below the function definition.
```

Comment:
Is this section necessary?
Contents seem to be duplicated with `Disabling paredit` section.

Reply:
github is rejecting my push with an "Internal Server Error".  I'll try again later.

Reply:
Sorry about the mess in the commits.  I tried to clean things up with rebase, but was not entirely successful.  If you want I can try something more radical to clean it up.

Reply:
Hmm, if we were to leave this section, the section title "Miscellaneous" seems inappropriate to me since contents are vim-sexp specific.

Personally, I don't want to duplicate the content of the document, so it is better for me to eliminate duplicate contents by using links to "Disabling paredit", etc.

> If you want I can try something more radical to clean it up.

No problem :)

Reply:
If you're seeing the title "Miscellaneous" then I've done something wrong.  Are you seeing the `paredit-tip` branch?  I'll look at this again later.

Reply:
Sorry about that!  I had moved the paredit section up after "Overwriting mappings", but I forgot to delete the old "Miscellaneous" version that I'd accidentally left at the end of the file.  The newest commit fixes this.  

Happy to change other things.